helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -174,38 +174,44 @@ spec:
     # -- error recording rules
     # record when pods of a daemonset with label "label_giantswarm_io_monitoring_basic_sli" are down
     - expr: |
-        kube_daemonset_labels * on (daemonset, namespace) group_right(label_application_giantswarm_io_team) (
-          label_replace(
+        label_replace(
```

Comment:
Removed the extra join as a group_left in the inner condition is enough to get what we want :)